src/js/components/Tab/Tab.js

```diff
@@ -114,9 +146,6 @@ let TabDoc;
 if (process.env.NODE_ENV !== 'production') {
   TabDoc = require('./doc').doc(Tab); // eslint-disable-line global-require
 }
-const TabWrapper = compose(
-  withTheme,
-  withForwardRef,
-)(TabDoc || Tab);
+const TabWrapper = compose(withTheme, withForwardRef)(TabDoc || Tab);
```

Comment:
As long as we're here, can we use `const theme = withContext(ThemeContext);` and also just wrap with React's `forwardRef()`? That allows us to remove `compose()`.

Reply:
Yes, let me look at that

Reply:
I've implemented this @ericsoderberghp, can you confirm it's been done correctly? storybooks functioned the same and no snapshots changed so I think it is correcy.